src/test/miner_tests.cpp

```diff
@@ -501,8 +501,11 @@ BOOST_AUTO_TEST_CASE(CreateNewBlock_validity)
     // For now these will still generate a valid template until BIP68 soft fork
     BOOST_CHECK_EQUAL(pblocktemplate->block.vtx.size(), 3U);
     // However if we advance height by 1 and time by 512, all of them should be mined
```

Comment:
```suggestion
    // However if we advance height by 1 and time by SEQUENCE_LOCK_TIME, all of them should be mined
```
https://github.com/bitcoin/bitcoin/pull/24804#issuecomment-1092630122

Reply:
Okay